source/Tentacle.sln

```diff
@@ -67,16 +65,6 @@ Global
 		Debug|Any CPU = Debug|Any CPU
 		Release|Any CPU = Release|Any CPU
 		Release-net48-win|Any CPU = Release-net48-win|Any CPU
-		Release-net6.0-linux-arm|Any CPU = Release-net6.0-linux-arm|Any CPU
```

Comment:
Just verifying these were redundant so don't need replacing with net8.0 ones?

Reply:
We already have net8.0 equivalents. This removes the net6.0 ones.